src/OrbitGl/MockBatcher.cpp

```diff
@@ -79,22 +81,22 @@ int MockBatcher::GetNumBoxes() const {
 // To check that everything is inside a rectangle, we just need to check the minimum and maximum
 // used coordinates.
 bool MockBatcher::IsEverythingInsideRectangle(Vec2 start, Vec2 size) const {
-  return IsInsideRectangle(Vec3ToVec2(min_point_), start, size) &&
-         IsInsideRectangle(Vec3ToVec2(max_point_), start, size);
+  if (!GetNumElements()) return true;
```

Comment:
I think we agreed to not use implicit casting to bool, so this should be `if (GetNumElements() == 0) return true;`